src/biokbase/narrative/jobs/util.py

```diff
@@ -1,3 +1,57 @@
+import os
+import json
+
+JOB_CONFIG_FILE_PATH_PARTS = [
+    "kbase-extension",
+    "static",
+    "kbase",
+    "config",
+    "job_config.json",
+]
+
+
+def load_job_constants(relative_path_to_file=JOB_CONFIG_FILE_PATH_PARTS):
```

Comment:
load up the constants and run some checks to make sure that we have all the message and param names that we need